dependencies.properties

```diff
@@ -62,6 +62,7 @@ maven.com_google_guava_failureaccess=com.google.guava:failureaccess:1.0.1
 maven.junit_junit=junit:junit:4.12
 maven.org_mockito_mockito_core=org.mockito:mockito-core:2.21.0
 maven.org_hamcrest_hamcrest_core=org.hamcrest:hamcrest-core:1.3
+maven.com_github_stefanbirkner_system_rules=com.github.stefanbirkner:system-rules:1.19.0
```

Comment:
We need to verify if the "Common Public License Version 1.0" is a license we're allowed to use.

Reply:
Let's skip adding the third_party dependency and inject a non-public EnvironmentProvider which can be removed later.

Reply:
Turns out we are not able to use this plugin as its license is not acceptable.